webpack.config.js

```diff
@@ -4,6 +4,7 @@ module.exports = buildJSFiles([
     'plugins/plugin.archive_analytics.js',
     'plugins/menu_toggle/plugin.menu_toggle.js',
     'plugins/tts/plugin.tts.js',
+    'plugins/plugin.autoplay.js'
```

Comment:
Let's keep these sorted; that way we'll have fewer merge conflicts than if we always added to the end :P